ts/nni_manager/config/dlc/dlcUtil.py

```diff
@@ -41,6 +43,13 @@
         data_source_id=args.nas_data_source_id,
     )
 
+    oss = None
+    if (args.oss_data_source_id):
+        oss = DataSourceItem(
+            data_source_type = 'oss',
+            data_source_id=args.oss_data_source_id,
```

Comment:
Please make spaces around `=` consistent in these two lines.

Reply:
accept, changed.